deps/tools/jldownload

```diff
@@ -10,9 +10,9 @@ CURL=$(which curl 2>/dev/null)
 FETCH=$(which fetch 2>/dev/null)
 
 TIMEOUT=15 # seconds
-WGET_OPTS="--no-check-certificate --tries=1 --timeout=$TIMEOUT"
-CURL_OPTS="-fkL --connect-timeout $TIMEOUT -y $TIMEOUT"
-FETCH_OPTS="-T $TIMEOUT"
+WGET_OPTS="-nv --no-check-certificate --no-use-server-timestamps --tries=1 --timeout=$TIMEOUT"
+CURL_OPTS="-fkLSs --connect-timeout $TIMEOUT -y $TIMEOUT"
```

Comment:
I'm 70% against making these completely silent; on slow connections it may look like the build is needlessly frozen.

Reply:
we already silence much of what happens in deps. I contemplated making it conditional on the presence of `--jobserver-fds=` (or `-j`) in `MAKEFLAGS`, but didn't seem worth it

Reply:
Most of what we silence doesn't depend on network speed though; A users build could potentially freeze for 2-3 minutes due to being stuck downloading an LLVM build; I think we should continue to print out download status reports to stdout.